# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/quick-lounge-applet/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual gcc-3.4 compile fixes.

  - Rene Rebe <rene@exactcode.de>

--- ./src/file-utils.c.orig	2004-05-13 21:17:57.915157080 +0200
+++ ./src/file-utils.c	2004-05-13 21:18:02.694430520 +0200
@@ -137,6 +137,7 @@
 			break;
 
 		default:
+			;
 		}
 
 		if (full_uri)
